draft-ietf-quic-recovery.md

```diff
@@ -463,9 +466,9 @@ Pseudocode for OnLossDetectionAlarm follows:
 ## Detecting Lost Packets
 
 Packets in QUIC are only considered lost once a larger packet number is
-acknowledged.  DetectLostPackets is called every time there is a new largest
-packet or if the loss detection alarm fires the previous largest acked packet is
-supplied.
+acknowledged.  DetectLostPackets is called every time an ack is received.
+If the loss detection alarm fires and the lost_time is set, the previous
```

Comment:
loss_time is somewhat orthogonal to reordering theshold.  Do you mean reordering_threshold and time_reordering_fraction, which are more analagous?